bxbot-tests/Services/PairingServiceTests.cs

```diff
@@ -31,6 +32,14 @@ public class PairingServiceTests
             var pairingList = (List<Pairing>)pairings;
             pairingList.Count.Should().Be(length, "the data returned is such.");
 
+            // This is to force fail a build
+            if(this != null)
+            {
+                // Skip
+            } if(this != null)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move this 'if' to a new line or add the missing 'else'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3972)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to 'true'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS2589)